src/config.ts

```diff
@@ -22,6 +22,14 @@ export const mainConfig = {
 		dex: 0.0,
 		// below is an extra manual transfer from the F-AERO multisig to the DEX relayer
 		dexRelayerAmount: 600_191.93879185, // 7,202,303.2655022416 WELL / 12 4-week epochs
```

Comment:
These should be zero now that the 2 year Aerodrome incentive campaign has ended.